README.md

```diff
@@ -8,19 +8,38 @@ Join the discord server for discussing and developing this tool [here](https://d
 
 1. Create a .env file with the following variables: 
 
-    ```shell
-    DISCORD_TOKEN=
-    OPENAI_API_KEY=
-    REDIS_URL=  # Optional
-    CLIENT_ID=
-    GUILD_ID=
-    SUMMARIZE_FREQUENCY_SECONDS=3600
-    ```
+```shell
+DISCORD_TOKEN=
+OPENAI_API_KEY=
+REDIS_URL=
+CLIENT_ID=
+GUILD_ID=
+SUMMARIZE_FREQUENCY_SECONDS=3600
+```
```

Comment:
if shell example is indented to be displayed under 1st step, then removing indentation breaks the alignment